### ISSUE TITLE
Agda kumite says theres failed tests while actually no

### ISSUE THREAD

**Issue by ice1000** [Codewars/codewars-runner-cli#723]
#### Describe the bug

Kumite: https://www.codewars.com/kumite/5c86255c5387ef000cc7625c/edit by @lwoo1999

#### To Reproduce

Click publish on the kumite, it passes typecheck but says there're failed tests.

#### Expected Behavior

Successfully published

#### Screenshots

![image](https://user-images.githubusercontent.com/16398479/54126352-b41acf00-43dd-11e9-9838-b87df98dab89.png)

**Comment by ice1000** [comment]
Problem solved: @lwoo1999 need to do the publish, not me.

But the error message should be changed, I believe.

**Comment by kazk** [comment]
I don't think this is specific to Agda and I honestly have no idea at the moment. Thanks for reporting.

Were you trying to publish a unpublished Kumite by someone else?

**Comment by ice1000** [comment]
> Were you trying to publish a unpublished Kumite by someone else?

Exactly.

**Comment by kazk** [comment]
Did it actually publish? Or just showed that error and nothing happened?
